docs/migrate-from-emby.md

```diff
@@ -0,0 +1,72 @@
+---
+id: admin-migrate-from-emby
+title: Migrating from Emby
```

Comment:
This name is originally from when migrating from emby was possible and common. perhaps we should rename it to just Migrations. We could also put it under the Server section (and remove off the sidebar) since these methods aren't technically supported and aren't guaranteed to work.

Reply:
Maybe "Migrating to Jellyfin", with a warning that it's possibly deprecated?

Reply:
i was thinking that there's also cross system migrations like baremetal > docker migrations that we should provide documentation on. "Migrations" should be vague enough

Reply:
Fair, that makes sense.

![image](https://user-images.githubusercontent.com/19396809/117554065-9ac14c00-b055-11eb-8ddc-fddc3c60f78d.png)

I moved it to "Tips & tricks" (For which I'd like to find a different name... If you have ideas x)